assets/images/qrcode.svg

```diff
@@ -1,5 +1,5 @@
-<svg width="20" height="20" viewBox="0 0 15 14" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
@sobitneupane after removing `fill="#03D47C"` the fix work fine, but if I open the image if Finder with any image viewer app it shows without any color, so I delete `fill="none"` also to display in finder with black color like other images.